static/css/main.css

```diff
@@ -2438,6 +2439,12 @@ body {
   white-space: nowrap;
 }
 
+@media screen and (max-width: 980px) {
+  #header .main ul li {
+    border-bottom: solid 1px rgba(160, 160, 160, 0.3);
+  }
+}
+
```

Comment:
Je me sens pas vraiment qualifié pour reviewer du CSS et je conais pas vraiment le codebase du site web donc peut-être ce que je vais dire est out-of-place. Cela étant dit, ce bout de code me semble un peu hackish. J'ai regardé rapidement dans les developer tools de Firefox et je me suis aperçu qu'en désactivant le `background-color: inherit;` [ìci](https://github.com/dot-layer/blog/pull/135/files#diff-7c6d9b791479470684ffe55c750a991e18239247dc75846ca2e688606e8e8887L2534), ça avait l'air de régler le problème. D'ailleurs, je ne vois pas trop pourquoi on a un `position: absolute` là mais c'est peut-être une discussion pour une autre fois. Qu'est-ce que tu en penses?

Reply:
Freeedddd! Allo! 
Ca marche en theorie dans le browser, mais dans le code ca chizzzz... Je mets la faute sur le fait que c'est un fichier css avec 3000 lignes plein de id... J'essaie de pas trop toucher au css parce que juste enlever le position absolute dont tu parles pete le sidenav haha
Pourquoi absolute? Aucune idee, je referais toute le css en scss en flexbox haha

Reply:
Anniiiiee! Yo! Haha. Je vois. Ça semble être honnête dans ce cas.

Reply:
Je suis toujours honnete haha <3